app/views/practices/_search.js.erb

```diff
@@ -86,67 +86,75 @@ function buildPracticeCard(result) {
     var cardHtml = '';
     var practiceImage = result.item.image;
     var practiceName = result.item.name;
-    var practiceLinkClasses = practiceImage ? 'dm-practice-link' : 'text-white text-no-underline';
-    var cardHeaderClasses = practiceImage ? ' margin-top-0 margin-bottom-2' : ' margin-x-2 margin-y-4';
-    var cardImageHtml =
-        '<div class="dm-practice-card-img-container">' +
-        '<input type="hidden" value="' + practiceImage + '"/>' +
-        '<img ' +
-          'data-resource-id="' + result.item.id + '" ' +
-          'alt="' + practiceName + ' Marketplace Card Image" ' +
-          'class="grid-row marketplace-card-img radius-top-sm" ' +
-          'src="' + result.item.placeholder_image + '">';
-    var retiredBanner = '<div class="dm-practice-retired-banner"><h5>Retired</h5></div>';
-
-    if (result.item.retired) {
-        cardImageHtml += retiredBanner;
-    }
+    var practiceId = result.item.id;
 
     var cardHeaderHtml =
-        '<h3 class="text-normal font-sans-lg multiline-ellipses-2' + cardHeaderClasses + '">' +
-        '<a href="/innovations/' + result.item.slug + '" aria-label="Go to ' + result.item.name + '" class="' + practiceLinkClasses +'">' +
-        '<span class="dm-practice-title">' +
-        practiceName +
-        '</span>' +
-        '</a>' +
+        '<h3 class="text-normal font-sans-lg multiline-ellipses-2 margin-top-0 margin-bottom-2">' +
+            '<span class="dm-practice-title">' +
+                practiceName +
+            '</span>' +
         '</h3>';
 
-    cardImageHtml += '</div>';
-
-    var noCardImageHtml =
-        '<div class="dm-bg-gray-80">' +
-        cardHeaderHtml;
+    // Default card image
+    var cardImagePlaceholder = 
+        '<div class="dm-bg-gray-80" data-practice-id="' + practiceId + '" data-practice-image="' + practiceImage + '"></div>'
```

Comment:
```suggestion
        '<div class="practice-card-img-placeholder" data-practice-id="' + practiceId + '" data-practice-image="' + practiceImage + '"></div>'
```
Since this CSS class is specific to practice cards, let's use a more specific CSS name